### PR TITLE
profile user and company

### PR DESCRIPTION
**Pull request by quanglx-noovado**
## Related Tickets:
- [#32126](https://edu-redmine.sun-asterisk.vn/issues/32126)

## WHAT this PR does?
View user profile and company info
comany_info
user_profile

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by quanglx-noovado** [comment]
ready

**Comment by quanglx-noovado** [comment]
ready

**Comment by quanglx-noovado** [comment]
ready

File: app/Http/Controllers/CompanyController.php [hunk 1]
```diff
@@ -49,7 +49,10 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $company = Company::find($id);
+        $images= $company->images()->where('type',config('user.avatar'))->first();
```
Comment (huyenttm-0901) [inline app/Http/Controllers/CompanyController.php]:
```suggestion
        $images = $company->images()->where('type',config('user.avatar'))->first();
```

File: app/Http/Controllers/CompanyController.php [hunk 2]
```diff
@@ -83,13 +86,13 @@ public function update(UpdateCompanyRequest $request, $id)
             if (isset($avatar) && ($avatar->getMimeType() === config('user.png') || $avatar->getMimeType() === config('user.jpg'))) {
                 $urlAvt = $avatar->move(public_path() . config('user.upload_company'), date(config('user.date_time')) . $avatar->getClientOriginalName());
                 $image = $company->images()->where('type', config('user.avatar'))->first();;
-
+                $url = config('user.upload_company').date(config('user.date_time')). $avatar->getClientOriginalName();
```
Comment (huyenttm-0901) [inline app/Http/Controllers/CompanyController.php]: convention

File: resources/views/user_profile.blade.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
+@extends('layouts.app')
+
+@section('css')
+    <link href="{{ asset('css/user.css') }}" rel='stylesheet' type='text/css' />
+@endsection
+@section('content')
+    <div class="container-1">
+        <div class="profile-header">
+            <div class="profile-img">
+                <img src="{{ asset($user->image->url) }}" width="200" alt="Profile Image">
```
Comment (huyenttm-0901) [inline resources/views/user_profile.blade.php]: i18n

**Comment by quanglx-noovado** [comment]
ready

**Comment by quanglx-noovado** [comment]
ready

**Comment by quanglx-noovado** [comment]
ready

**Pull request merged**